actions/image_partition_action.go

```diff
@@ -145,6 +149,7 @@ import (
 	"fmt"
 	"github.com/docker/go-units"
 	"github.com/go-debos/fakemachine"
+	"github.com/google/uuid"
```

Comment:
what's the uuid lib used for ?

Reply:
right, to check if the uuid is correct. Can't seem to resolve these comments for some reason :(

Reply:
Sorry for the delay.

The uuid library is used for verifying the input value of the FSUUID property (specified by the user on the image-partition action). The verification is based on RFC 4122 and DCE 1.1: Authentication and Security Services. Check
https://github.com/google/uuid for more details.

I have added this information to the commit as well, just in case someone wants to know while looking at the commit log.